common/source/docs/common-measuring-vibration.rst

```diff
@@ -72,6 +72,16 @@ the accelerometer readings like this:
    square root (for x, y and z).  These final three values are what
    appear in the VIBE msg's VibeX, Y and Z fields.
 
+When looking at vibrations in a log the first thing to look at is the clipping. If the clipping is 0 that’s good. That means the vibrations that are being detected aren’t overwhelming the IMU.
+
+When troubleshooting the vibration, consider the axis of the vibration as a staring point to find the problem:
+
+    If X AND Y are both high, then you may have an issue with a motor bearing or prop balance. Or you may need more/better overall vibration damping for your FC.
+    If X OR Y is high then you may have an issue with your FC mounting. Maybe a wire or is bouncing on the FC or restraining it. Or you maybe your vibration damping works better one way than the other.
+    If you have a Z vibration then you may have a track issue with propeller (bent blade) or vertical play in a motor.
+
+Also consider that some flight conditions/airframes will have different natural vibrations. If the vibrations look good in a hover,  but they goes up with speed, perhaps there is an aerodynamic issue on the airframe, or the wind,
```

Comment:
```suggestion
Also consider that some flight conditions/airframes will have different natural vibrations. If the vibrations look good in a hover, but they goes up with speed, perhaps there is an aerodynamic issue on the airframe, or the wind,
```

Reply:
Josh...tweaked it a bit

Reply:
looks good  to me ! Thanks!

Reply:
Thank!  This is my first PR so I'm not sure if there's anything I need to do now? (or have access to do?)